islandora_transforms/slurp_all_chemicalML_to_solr.xslt

```diff
@@ -16,76 +16,78 @@
     <xsl:param name="suffix"></xsl:param>
     <!-- Clearing hash in case the template is ran more than once. -->
     <xsl:variable name="return_from_clear" select="java:clear($single_valued_hashset_for_cml)"/>
-    <xsl:apply-templates mode="writing_cml" select="$content/cmls:module"/>
+
+    <xsl:apply-templates mode="writing_cml" select="$content//cmls:module"/>
     <xsl:apply-templates mode="slurping_cml" select="$content//cmls:metadataList[@convention = 'islandora:sp_chem_CM']"/>
   </xsl:template>
 
   <!-- Match on individual fields. -->
-  <xsl:template match='cmls:module[@dictRef="cc:finalization"]/cmls:propertyList/cmls:property[@dictRef="cc:multipole"]/cmls:list/cmls:array[@dictRef="cc:hexadecapole"]' mode="writing_cml">
+
+  <xsl:template match='//cmls:module[@dictRef="cc:finalization"]/cmls:propertyList/cmls:property[@dictRef="cc:multipole"]/cmls:list/cmls:array[@dictRef="cc:hexadecapole"]' mode="writing_cml">
```

Comment:
These changes seem to be a regression of https://github.com/discoverygarden/islandora_transforms/commit/29371333e59ec6edc96a13872dc2c4e1fbc9c00b